src/components/EnterBarcode/EnterBarcodePage.tsx

```diff
@@ -23,22 +23,30 @@ const EnterBarcodePage = ({
   const handleEnterBarcode = () => {
     if (!loading && data) {
       const selectedPatient = data.patients.filter(
-        (patient) => patient.barcodeValue.toString() === barcode
+        (patient) => patient.barcodeValue === barcode
       );
       if (selectedPatient.length > 0) {
         // Found patient
         const {
-          collectionPointId: { id: patientCCPId },
+          collectionPointId: {
+            id: patientCCPId,
```

Comment:
to fix the build issue
```suggestion
            id: patientCcpId,
```